include/experimental/__p0009_bits/mdspan.hpp

```diff
@@ -55,6 +55,13 @@ class mdspan
     ReferenceType __callop(mdspan const& __self, const std::array<SizeType, N>& indices) noexcept {
       return __self.__accessor_ref().access(__self.__ptr_ref(), __self.__mapping_ref()(indices[Idxs]...));
     }
+#ifdef __cpp_lib_span
+    template <class ReferenceType, class SizeType, size_t N>
+    MDSPAN_FORCE_INLINE_FUNCTION static constexpr
+    ReferenceType __callop(mdspan const& __self, const std::span<SizeType, N>& indices) noexcept {
+      return __self.__accessor_ref().access(__self.__ptr_ref(), __self.__mapping_ref()(indices[Idxs]...));
+    }
+#endif
```

Comment:
This was just missing, even though the mdspan ctor called it. We just never tested it before ...

Reply:
I'm confused -- wouldn't the be operator[] calling it? The mdspan.index_operator.pass.cpp test does call it, so I'm not entirely sure why it was being instantiated

EDIT: oh that test was disabled before this PR